index.markdown

```diff
@@ -5,23 +5,18 @@ date: 2016-08-07 02:13:26+00:00
 layout: page
 slug: index
 title: index
+redirect_from: "/moveit/"
 ---
 
 MoveIt! is state of the art software for mobile manipulation, incorporating the latest advances in motion planning, manipulation, 3D perception, kinematics, control and navigation. It provides an easy-to-use platform for developing advanced robotics applications, evaluating new robot designs and building integrated robotics products for industrial, commercial, R&D and other domains.
 
 MoveIt! is the most widely used open-source software for manipulation and has been used on over 65 robots. See the [Robots](/robots/) page for a list of some of the robots that MoveIt! is being used with.
 
-
 <p>
   <iframe src="//www.youtube.com/embed/dblCGZzeUqs" frameborder="0" width="560" height="315">
   </iframe>
 </p>
 
-
-
-
-**Need Custom Development?**: If you need help with integration of MoveIt! on your robot or applications, please email _robot_ dot _moveit_ at _gmail_ dot _com_
-
```

Comment:
I don't think it's a good idea to remove this paragraph. Instead I would propose to refer people to the moveit-users mailing list if they encounter problems.